pkg/keyspace/keyspace.go

```diff
@@ -148,6 +171,16 @@ func (manager *Manager) CreateKeyspace(request *CreateKeyspaceRequest) (*keyspac
 	if err != nil {
 		return nil, err
 	}
+	userKind := endpoint.StringUserKind(request.Config[UserKindKey])
+	id, err := manager.kgm.GetAvailableKeyspaceGroupIDByKind(userKind)
+	if err != nil {
+		return nil, err
+	}
+	if request.Config == nil {
```

Comment:
Why do we do the nil check right after we try to get a key from it at line 174?

Reply:
The nil map can be accessed but cannot be assigned.